src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <body>
+      <header class="header">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
```

Comment:
let's think of more descriptive and clear naming for the `alt` attribute. It should be clear exactly what is it, it is an alternate description

Reply:
is it better now? https://prnt.sc/WGED1UItn3en